service/src/main/kotlin/app/cash/backfila/service/runner/BackfillRunner.kt

```diff
@@ -184,7 +195,10 @@ class BackfillRunner private constructor(
   ): Deferred<RunBatchResponse> {
     // Supervisor here allows us to handle the exception, rather than failing the job.
     return scope.async(SupervisorJob()) {
-      client.runBatch(RunBatchRequest(
+      val callStartedAt = factory.clock.instant()
```

Comment:
Do we want to use stopwatches instead?
https://guava.dev/releases/18.0/api/docs/com/google/common/base/Stopwatch.html
I use that in the Dynamo client.